packages/f36-docs-utils/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@contentful/f36-docs-utils",
+  "version": "4.0.0",
+  "private": true,
+  "license": "MIT",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "main": "dist/main.js",
+  "types": "dist/types.d.ts",
+  "source": "src/index.ts",
+  "sideEffects": false,
+  "browserslist": "extends @contentful/browserslist-config",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/contentful/forma-36"
+  },
+  "scripts": {
+    "build": "parcel build index.ts",
+    "watch": "parcel watch index.ts"
+  },
+  "dependencies": {
+    "constate": "^3.2.0",
+    "@babel/runtime": "^7.6.2",
+    "@contentful/f36-tokens": "^4.0.0",
+    "@contentful/f36-components": "^4.0.0",
+    "lodash.orderby": "^4.6.0"
```

Comment:
This can go in a different PR really, but we might as well think about it here. These separate lodash method packages aren't being updated, not even for security issues. This one is 5 years old. We should just use the main package with the webpack plugin instead.

Reply:
this package is used only on the documentation website, so we can use lodash as it is

Reply:
And the timepicker 🙂

Reply:
yeah, but this particular package is used only on the documentation website, so I'll just replace it here without touching timepicker

Reply:
updated